corehq/apps/builds/views.py

```diff
@@ -20,6 +19,7 @@
 from .models import CommCareBuild, CommCareBuildConfig, SemanticVersionProperty
 from .utils import get_all_versions, extract_build_info_from_filename
 
+import io
```

Comment:
I think io is a standard library import that should go at the top right? (looks like this files imports are kind of wonky anyways)

Reply:
I put this import with the other standard library imports because I figure it's best to keep things organized even if it's not pep8-y.  If someone else wants to re-order the imports it'll be easier if there is some organization, even if it's not pep8-y.